src/mod/irc.mod/chan.c

```diff
@@ -1811,6 +1823,16 @@ static int gotjoin(char *from, char *chname)
         m->user = u;
         m->flags |= STOPWHO;
 
+        if (extended_join) {
```

Comment:
this is dead code, right? now that check_tcl_account is removed this does nothing

Reply:
some of it... ;)